File: Ruby/Euler020.rb
#!/usr/bin/env ruby

require_relative 'Euler015.rb'

def main
  target = 100
  result = factorial(target)
  sum = 0

  while result > 0
    result, digit = result.divmod(10)
    sum += digit
  end

  puts sum
end

if __FILE__ == $0
  main
end

File: Python/Euler020.py
#!/usr/bin/env python3
from math import factorial


def main():
    target = 100
    result = factorial(target)
    s = 0

    while result > 0:
        result, digit = divmod(result, 10)
        s += digit

    print(s)


if __name__ == '__main__':
    main()
